### ISSUE TITLE
html tag selector produces errors

### ISSUE THREAD

**Issue by CubeOfCheese** [stylelint/stylelint#5319]
using "html" as a selector produces multiple stylelint errors using the standard configuration.

> Which rule, if any, is the bug related to?

Unknown rule declaration-block-no-duplicate-custom-properties.
Unknown rule named-grid-areas-no-invalid.
Unknown rule no-invalid-position-at-import-rule.
Unknown rule no-irregular-whitespace.

> What code is needed to reproduce the bug?

```
html {
  margin: 0;
  height: 100%;
}
```

> What stylelint configuration is needed to reproduce the bug?

standard configuration

> Which version of stylelint are you using?

13.13.1

> How are you running stylelint: CLI, PostCSS plugin, Node.js API?

I am using the linter-stylelint plugin for atom https://atom.io/packages/linter-stylelint

> Does the bug relate to non-standard syntax (e.g. SCSS, Less etc.)?

no

> What did you expect to happen?

No errors to be flagged.

> What actually happened (e.g. what warnings or errors did you get)?

e.g. "The following warnings were flagged:"

Unknown rule declaration-block-no-duplicate-custom-properties.
Unknown rule named-grid-areas-no-invalid.
Unknown rule no-invalid-position-at-import-rule.
Unknown rule no-irregular-whitespace.

all on line 1 column 1

**Comment by jeddy3** [comment]
Thanks for the report and for using the template.

These rules were [recently added to stylelint](https://github.com/AtomLinter/linter-stylelint). It seems the atom linter [bundles an older version of stylelint](https://github.com/AtomLinter/linter-stylelint/blob/master/package.json#L64). I believe the atom linter will use a locally installed copy of stylelint if it is present. So, you can do `npm i -D stylelint@latest` to install it.
